sgf-viewer/src/content.ts

```diff
@@ -173,7 +173,8 @@ export const pages: object =
                 "dir_name": "amateur_vs_advh_497mil",
                 "server": "dqn.ist.berkeley.edu",
                 "paths": [
-                    "/nas/ucb/ttseng/go_attack/misc/tony-vs-dragonslayer-497mil.sgfs"
+                    "/nas/ucb/ttseng/go_attack/misc/tony-b-def-dragonslayer-s497mil-w-anon.sgfs",
+                    "/nas/ucb/ttseng/go_attack/misc/tony-w-def-dragonslayer-s497mil-b-anon.sgfs"
```

Comment:
note for Tony: I couldn't use `/nas/ucb/tony/go-attack/manual-games/tony-b-def-dragonslayer-s497mil-w-anon.sgf` directly since the adversary name was accidentally put in `WR[]` rather than `PW[]`, and the game parser didn't like that. And I (reasonably) don't have permissions to modify things in `/nas/ucb/tony`

Reply:
Ah no problem, this was my bad.